amethyst_network/src/server/udp.rs

```diff
@@ -0,0 +1,136 @@
+//! All UDP related logic for getting and sending data out to the other side.
+
+use crate::{
+    error::Result,
+    server::{
+        ClientEvent, PacketReceiving, PacketSending, ReceiveHandler, SendHandler, ServerConfig,
+        ServerSocketEvent,
+    },
+};
+use laminar::net::UdpSocket;
+use laminar::NetworkConfig;
+use std::net::SocketAddr;
+use std::sync::mpsc::{self, Receiver, Sender};
+use std::thread;
+
+/// An UDP receiver, wrapper for starting the UDP-receiving thread.
+pub struct UdpReceiver {
+    // socket used for receiving packets
+    socket: UdpSocket,
+    pub max_troughput: u16,
```

Comment:
This either needs a comment or if this can not change during the life of a UdpReceiver then I would move this to a `max_throughput()` function.